addons/stock/views/stock_move_views.xml

```diff
@@ -9,7 +9,7 @@
                     <field name="product_id"/>
                     <field name="product_uom_qty"/>
                     <field name="product_uom" options="{'no_open': True, 'no_create': True}" string="Unit of Measure" groups="product.group_uom"/>
-                    <field name="product_packaging" domain="[('product_tmpl_id','=',product_tmpl_id)]" groups="product.group_stock_packaging"/>
```

Comment:
You should replace it by domain="[('product_id','=',product_id)]" instead of remove it